README.md

```diff
@@ -39,6 +39,7 @@ For Arty A7-35T it would be [Vivado](https://www.xilinx.com/support/download.htm
 You always need a toolchain for a target that you choose.
 
 If you want to try things out using [Renode](https://renode.io) simulation, then you don't need either the board or toolchain.
+To install it follow the [Renode guidelines](https://github.com/renode/renode#installation) or run the `setup` script to get Renode Portable.
```

Comment:
@robertszczepanski  please update this, as we don't really support using installed Renode right now

Reply:
Sure, removed that line.